browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2067,7 +2067,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		}
 
 		controls['container'] = div;
-		div.tabIndex = -1;
```

Comment:
with it you get 2 times focus on the same button (check in sidebar / notebookbar)

Reply:
I think it works correctly and alread catches focus on `button` defined below

Reply:
> with it you get 2 times focus on the same button (check in sidebar / notebookbar)

No, I couldn’t reproduce it in the combobox; it jumps to the next focusable element.

Reply:
It is used in "Toolbuttons" -> see toolbar in compact mode, or in sidebar.
Regular "icon" buttons, not combobox.

Reply:
Unfortunately, "div" is focusable without setting tab index to "-1". I confirm that it requires 2 tab presses to move to next button.